doc/exporter-guide.md

```diff
@@ -0,0 +1,49 @@
+# Exporter Developer Guide
+
+An exporter sends traces and metrics to any backend that is capable of consuming them. The exporter itself can change without requiring a change in your client code.
+
+OpenTelemetry exporters can be contributed by anyone, and we provide support for several open source backends and vendors out-of-the-box, example Zipkin, Jaeger, Prometheus. This document will provide a means for developers to create their own. We strongly recommended to create a dedicated package for newly added exporter, example: `@opentelemetry/exporter-myexporter`.
+
+## Tracing
+
+`SpanExporter` defines the interface that protocol-specific exporters must implement so that they can be plugged into OpenTelemetry SDK and support sending of spans data. Exporters should expect to receive only sampled-in ended spans. Exporters must not throw. Exporters should not modify spans they receive.
```

Comment:
```suggestion
The `SpanExporter` interface defines which methods the protocol-specific trace/span exporters must implement so that they can be plugged into OpenTelemetry SDK. Span exporters must follow these rules:

1. Implement the `SpanExporter` interface.
2. Expect to only receive spans which have been sampled.
3. Expect to only receive spans which are ended.
4. Do not throw exceptions.
5. Do not modify received spans.
6. Do not implement queueing or batching logic because this is handled by Span Processors.
```